blockware/fireworks/lib/Particle/Particle.h

```diff
@@ -44,6 +46,8 @@ class Particle {
   float _resistance;
 
 protected:
+public:
+  static void update_gravity(const Vec2d<float> &gravity_xy);
```

Comment:
Minor nit and sorry I didn't catch this the first time. I think we use `camelCase` most places instead of `snake_case` though perhaps we're not super disciplined on this

Reply:
Yup, sorry :-P